optaplanner-core/src/test/java/org/optaplanner/core/impl/score/director/ScoreDirectorFactoryFactoryTest.java

```diff
@@ -140,16 +140,20 @@ void multipleScoreCalculations_throwsException() {
 
     @Test
     void nonExistingDrlResource_throwsException() {
-        ScoreDirectorFactoryConfig config = new ScoreDirectorFactoryConfig().withScoreDrls("nonExisting.drl");
-        assertThatExceptionOfType(IllegalArgumentException.class).isThrownBy(() -> buildTestdataScoreDirectoryFactory(config))
-                .withMessageContaining("scoreDrl");
+        ScoreDirectorFactoryConfig config = new ScoreDirectorFactoryConfig()
+                .withScoreDrls("nonExisting.drl");
+        assertThatExceptionOfType(IllegalStateException.class)
+                .isThrownBy(() -> buildTestdataScoreDirectoryFactory(config))
+                .withMessageContaining("DRL");
     }
 
     @Test
     void nonExistingDrlFile_throwsException() {
-        ScoreDirectorFactoryConfig config = new ScoreDirectorFactoryConfig().withScoreDrlFiles(new File("nonExisting.drl"));
-        assertThatExceptionOfType(IllegalArgumentException.class).isThrownBy(() -> buildTestdataScoreDirectoryFactory(config))
-                .withMessageContaining("scoreDrlFile");
+        ScoreDirectorFactoryConfig config = new ScoreDirectorFactoryConfig()
+                .withScoreDrlFiles(new File("nonExisting.drl"));
+        assertThatExceptionOfType(IllegalStateException.class)
+                .isThrownBy(() -> buildTestdataScoreDirectoryFactory(config))
+                .withMessageContaining("DRL");
```

Comment:
I 'd argue it needs to have the word "scoreDRL" in the message, because that is the breadcrumb that ties back into the solverConfig's `<scoreDRL>` line.
It's also important for us when we get support tickets, to know that they went through scoreDrl and not though kmodule.xml or CS.

Reply:
This code will only be used when they go through scoreDrl/scoreDrlFile.
In fact, I'd argue that saying "DRL" is better than saying "scoreDrl or scoreDrlFile".